api/src/pcapi/core/finance/api.py

```diff
@@ -518,15 +518,25 @@ def _price_event(event: models.FinanceEvent) -> models.Pricing:
         ):
             new_revenue += event.bookingFinanceIncident.newTotalAmount
 
-    rule_finder = reimbursement.CustomRuleFinder()
-    rule = reimbursement.get_reimbursement_rule(booking, rule_finder, new_revenue)
-
-    if event.motive == models.FinanceEventMotive.INCIDENT_REVERSAL_OF_ORIGINAL_EVENT:
+    if is_incident_event:
         original_pricing = models.Pricing.query.filter_by(
             status=models.PricingStatus.INVOICED,
             booking=individual_booking,
             collectiveBooking=collective_booking,
         ).one()
+
+        rule = None
+        if original_pricing.customRuleId:
+            rule = models.CustomReimbursementRule.query.get(original_pricing.customRuleId)
+
+        rule_description = original_pricing.standardRule
```

Comment:
Question : doit-on vraiment prendre `standardRule` dans le cas `if original_pricing.customRuleId:` ?
(je n'en sais rien, c'est juste pour être sûr au cas où...)

Reply:
Dans le cas ou le pricing contient un `customRuleId`, alors son champ `standardRule` est à None dans tous les cas

Reply:
Ok, mais a priori ce n'est pas `None` car n'est pas `nullable` dans le modèle de `Pricing` : 
`standardRule: str = sqla.Column(sqla.Text, nullable=False)`
Je vois cependant le contrainte `reimbursement_rule_constraint_check` qui semble confirmer que c'est une chaîne vide. Je suis partisan d'un petit commentaire qui aiderait tout de même à la compréhension : 
```suggestion
        if original_pricing.customRuleId:
            rule = models.CustomReimbursementRule.query.get(original_pricing.customRuleId)

        rule_description = original_pricing.standardRule  # empty when customRuleId is set
```

Reply:
Ah oui autant pour moi, commentaire vide :+1: